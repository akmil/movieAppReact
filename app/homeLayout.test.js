import React from 'react';
// import ReactDOM from 'react-dom';
// import renderer from 'react-test-renderer';
import HomeLayout from './homeLayout';
import { shallow } from 'enzyme';

describe('HomeLayout', () => {
  it('should render without errors', () => {
    // const wrapper = renderer.create(<HomeLayout/>)
    const wrapper = shallow(<HomeLayout/>);
    expect(wrapper).toMatchSnapshot();
  });
});
