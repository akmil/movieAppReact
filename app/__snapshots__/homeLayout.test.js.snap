// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeLayout should render without errors 1`] = `
<div
  className="main-layout-wrapper"
>
  <Connect(withRouter(Header)) />
  <ResultsBox />
</div>
`;
